### DISCUSSION TITLE
How to get prev & next post using blade engine

### DISCUSSION THREAD

**Discussion by aknEvrnky** [statamic/cms#8521] [Q&A]
I've read the [document page](https://statamic.dev/tags/collection-previous). I want to get next and prev post if they exists. how can I get them using laravel blade engine?

What I tried is:
```php
$prev = Statamic::tag('collection:prev')->params([
      'in' => 'blogs',
      'current' => $page->id,
  ]);
```

**Comment by ryanmitchell** [comment]
Put a ->fetch() on the end of that and it should give you a result?
